data/presets/highway/busway.json

```diff
@@ -0,0 +1,32 @@
+{
+    "icon": "temaki-bus",
+    "fields": [
+        "name",
+        "operator",
+        "oneway",
+        "structure",
+        "covered"
+    ],
+    "moreFields": [
+        "trolley_wire"
+    ],
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "highway": "busway"
+    },
+    "addTags": {
+        "highway": "busway",
+        "access": "no",
+        "bus": "designated",
+        "emergency": "yes"
```

Comment:
why should these tags be added, given that they are implied by the [highway=busway](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dbusway) tag already?